mobify/platform-scaffold
# AMP-59: Footer

Add footer.

 **JIRA**: https://mobify.atlassian.net/browse/AMP-59

## Changes
- `<FieldRow>`, and  `<Field>` components
- Add Footer

## Todos
- [ ] Add amp-form for Subscribe to Newsletter

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](http://localhost:3000/)
- Scroll down to bottom to check out the Footer
